docs/tutorials/training-a-pointnav-model.md

```diff
@@ -514,25 +517,21 @@ python main.py -o <PATH_TO_OUTPUT> -c -b <BASE_DIRECTORY_OF_YOUR_EXPERIMENT> <EX
 ```
 If using the same configuration as we have set up, the following command should work:
 ```bash
-python main.py -o projects/tutorials/pointnav_robothor_rgb/storage/ -b projects/tutorials/pointnav_robothor_rgb/experiments pointnav_robothor_rgb_ddppo
+python main.py -o storage/robothor-pointnav-rgb-resnet-resnet -b projects/tutorials pointnav_robothor_rgb_ddppo
 ```
 If we start up a tensorboard server during training and specify that `output_dir=storage` the output should look
 something like this:
 ![tensorboard output](../img/tb.png)
```

Comment:
As we add more tutorials it will probably be a good idea to have more descriptive names than `tb.png` so that we can tell things apart.